Reorg security docs and add more WDAC content (#11416)
MicrosoftDocs/PowerShell-Docs: reference/docs-conceptual/security/security-features.md

## AMSI Support

> The Windows Antimalware Scan Interface (AMSI) is an API that allows application to pass actions to

Reviewer: ```suggestion
The Windows Antimalware Scan Interface (AMSI) is an API that allows applications to pass actions to
```

## Constrained language mode

> **ConstrainedLanguage** mode protects your system by limiting the cmdlets and .NET types that are

Reviewer: ```suggestion
**ConstrainedLanguage** mode protects your system by limiting the cmdlets and .NET types
```

allowed in a PowerShell session. For a full description, see [about_Language_Modes][04].

## Application Control

Windows 10 includes two technologies, [Windows Defender Application Control (WDAC)][08] and
[AppLocker][07] that you can use to control applications. PowerShell detects if a system wide
application control policy is being enforced. The policy applies certain behaviors when running
script blocks, script files, or loading module files to prevent arbitrary code execution on the
system.

WDAC is designed as a security feature under the servicing criteria defined by the Microsoft
Security Response Center (MSRC). WDAC is the preferred application control system for Windows.

For more information about how PowerShell supports AppLocker and WDAC, see
[Using Windows Defender Application Control][10].

## Software Bill of Materials (SBOM)

Beginning with PowerShell 7.2, all install packages contain a Software Bill of Materials (SBOM). The

> PowerShell team is also producing SBOMs for modules that they own but ship independently from

Reviewer: ```suggestion
PowerShell team also produces SBOMs for modules that they own but ship independently from
```

PowerShell.

You can find SBOM files in the following locations:

- In PowerShell, find the SBOM at `$PSHOME/_manifest/spdx_2.2/manifest.spdx.json`.
- For modules, find the SBOM in the module's folder under `_manifest/spdx_2.2/manifest.spdx.json`.

The creation and publishing of the SBOM is the first step to modernize Federal Government

> cybersecurity and enhance software supply chain security.For more information about this initiative,

Reviewer: ```suggestion
cybersecurity and enhance software supply chain security. For more information about this initiative,
```
